From daa3f535158f321e7a01ea531fc3085f2d26ab37 Mon Sep 17 00:00:00 2001
From: Hyunjin Song <tteu.ingog@gmail.com>
Date: Wed, 23 May 2018 14:59:45 +0900
Subject: [PATCH] Fix compilation with Qt 5.11 (#4374)

Add extends attribute for custom widget RowTableView per upstream uic change http://code.qt.io/cgit/qt/qtbase.git/commit/?id=058474884c2505a8a00d4c59b4922bfcd3597c2f
---
 src/gui/Forms/EffectSelectDialog.ui | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gui/Forms/EffectSelectDialog.ui b/src/gui/Forms/EffectSelectDialog.ui
index a9c6de0191..f8e7734862 100644
--- a/src/gui/Forms/EffectSelectDialog.ui
+++ b/src/gui/Forms/EffectSelectDialog.ui
@@ -102,6 +102,7 @@
  <customwidgets>
   <customwidget>
    <class>RowTableView</class>
+   <extends>QTableView</extends>
    <header>RowTableView.h</header>
   </customwidget>
  </customwidgets>
